docs/3_static.md

```diff
@@ -0,0 +1,178 @@
+exposed by GCP, so you can configure it however you see fit.
+
+Create a file named `cloudmemorystore.yaml` with the following content:
+
+```yaml
+apiVersion: cache.gcp.crossplane.io/v1beta1
+kind: CloudMemorystoreInstance
+metadata:
+  name: example-cloudmemorystore-instance
+spec:
+  providerRef:
+    name: gcp-provider
+  writeConnectionSecretToRef:
+    name: example-cloudmemorystore-connection-details
+    namespace: crossplane-system
+  reclaimPolicy: Delete
+  forProvider:
+    tier: STANDARD_HA
+    region: us-west2
+    memorySizeGb: 1
+```
+
+> *Note: there is no namespace defined on our configuration for the
+> `CloudMemorystoreInstance` above because it is
+> [cluster-scoped](https://kubernetes.io/docs/tasks/access-kubernetes-api/custom-resources/custom-resource-definitions/#create-a-customresourcedefinition).*
+
+Now, create a `CloudMemorystoreInstance` in your cluster with the following
+command:
+
+```
+kubectl apply -f cloudmemorystore.yaml
+```
+
+The GCP provider controllers will see the creation of this
+`CloudMemorystoreInstance` and subsequently create it on GCP. You can log in to
+the GCP console to view the the status of the resource, but Crossplane will also
+propagate the status back to the `CloudMemorystore` object itself. This allows
+you to manage your infrastructure without ever leaving `kubectl`.
+
+```
+kubectl describe -f cloudmemorystore.yaml
+```
+
+[TODO INSERT OUTPUT]
+
+When the resource is done provisioning on GCP, you should see the status turn to
+`Ready`, and a corresponding event will be emitted. At this point, Crossplane
+will create a `Secret` that contains any connection information for the external
+resource. The `Secret` is created in the location we specified in our
+configuration:
+
+```yaml
+writeConnectionSecretToRef:
+    name: example-cloudmemorystore-connection-details
+    namespace: crossplane-system
+```
+
+Take a look at the contents of the `Secret`:
+
+```
+kubectl -n crossplane-system describe secret example-cloudmemorystore-connection-details
+```
+
+You will also see that the `CloudMemorystoreInstance` resource is still
+reporting `Unbound`. This is because we have not *claimed* it for usage yet.
+
+Crossplane follows a similar pattern to [Kubernetes persistent
+volumes](https://kubernetes.io/docs/concepts/storage/persistent-volumes/). When
+you statically provision a resource in Crossplane, the external resource is also
+created. However, when you want to use a resource, you create an abstract
+**claim** for it. In this case, we will create a `RedisCluster` claim for the
+`CloudMemorystoreInstance`. Because we know exactly which
+`CloudMemorystoreInstance` we want to use, we reference it directly from the
+claim.
+
+> **Claim**: an instance of a namespace-scoped CRD that represents a claim for
+> usage of a managed resource. Claims are abstract types, like `RedisCluster`,
+> that have support across multiple providers. An abstract claim may be
```

Comment:
Nit: Should we highlight the application-focused nature of claims rather than their portability?

Reply:
Isn't it kind of true that the portability of claims is what makes this application-focused? I am not sure it is as clear here to say that claims are application-focused because it doesn't explain what they actually do. Do you have any suggestions on wording?